src/service/GameHandler.js

```diff
@@ -0,0 +1,36 @@
+
+export class GameHandler {
+    constructor(name, totalTime) {
+        this.name = name;
+        this.totalTime = totalTime; //seconds
+        this.score = 0;
+        this.answers = [];
+    }
+
+    addAnswer(correctAnswer, answer, isCorrect, question) {
+        this.answers.push({
+            answer: answer,
+            correctAnswer: correctAnswer,
+            isCorrect: isCorrect,
+            question: question,
+        })
+        if (isCorrect) {
+            this.score++;
+        }
+    }
+
+    // Subtruct time left from total time
```

Comment:
Dałoby radę zmienić nazwę metody na bardziej opisową i wtedy wywalić ten komentarz :) ? 
Komenatrze mogą się dezaktualizować i mówić nie to co naprawdę robi kod pod nimi.